libs/sdk-backend-tiger/src/backend/organization/settings.ts

```diff
@@ -83,4 +99,16 @@ export class OrganizationSettingsService
             }),
         );
     }
+
+    protected async deleteSettingByType(type: TigerSettingsType): Promise<void> {
+        const {
+            data: { data },
+        } = await this.getSettingByType(type);
+        const settingId = data.find((setting) => setting.attributes?.type === type)?.id;
```

Comment:
I know that this is unlikely but imho this should rather use `filter` than `find` and then delete all the settings that were found.